scripts/completion-tests/test-completion.sh

```diff
@@ -34,10 +34,26 @@ export -f set_shell_debug_level
 set_shell_debug_level 2
 
 BINARY_NAME=helm
-BINARY_ROOT=${ROBOT_HELM_PATH:-${SCRIPT_DIR}/../../../helm/bin}
-BINARY_PATH_DOCKER=${BINARY_ROOT}/../_dist/linux-amd64
+HELM_DIR=${SCRIPT_DIR}/../../.acceptance
+HELM_REPO=${HELM_DIR}/helm
+BINARY_ROOT=${HELM_REPO}/bin
+BINARY_PATH_DOCKER=${HELM_REPO}/_dist/linux-amd64
 BINARY_PATH_LOCAL=${BINARY_ROOT}
 
+if [ -d ${HELM_REPO} ]; then
+    (cd ${HELM_REPO} && make build build-cross) &> /dev/null
```

Comment:
It will be hard to understand what happened if the build fails. Would it make sense to let the output show on the console?

Reply:
I think it only sends stdout to null not stderr, so if the build fails it should output on console.

Reply:
Normally `>` is for stdout but `&>` is for both stdout and stderr.